deploy/namespace.yaml

```diff
@@ -6,3 +6,4 @@ metadata:
   labels:
     name: openshift-ptp
     openshift.io/run-level: "1"
+    openshift.io/cluster-monitoring: "true"
```

Comment:
Please also update the same for 4.4 manifest which is used when deploying ptp-operator via operator hub.
https://github.com/openshift/ptp-operator/blob/master/manifests/4.4/ptp-operator.v4.4.0.clusterserviceversion.yaml#L112